packages/react-magma-dom/src/components/Search/Search.test.js

```diff
@@ -106,6 +107,7 @@ describe('Search', () => {
         labelText={labelText}
         onSearch={onSearchSpy}
         onClear={onClear}
+        onDateChange={onDateChange}
```

Comment:
Search component shouldn't need `onDateChange` as search has nothing to do with dates

Reply:
fixed optional prop using